src/components/results/teacher/question-stats-table.tsx

```
@@ -0,0 +1,458 @@
+    questionText:
+      "Implement a binary search tree (BST) class with insert, search, and delete methods in your preferred programming language.",
+    questionType: "coding",
+    difficulty: "advanced",
+    averageScore: 5.2,
+    correctPercentage: 52,
+    attemptedCount: 6,
+    skippedCount: 4,
+  },
+];
+
+export function QuestionStatsTable({}: QuestionStatsTableProps) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)